scripts/lint-editorconfig.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+ec .github contrib docs package scripts targets tests ./*.* .luacheckrc .editorconfig
```

Comment:
```suggestion
editorconfig-checker .github contrib docs package scripts targets tests ./*.* .luacheckrc .editorconfig
```

Reply:
resolved in c93eb80bbe44f290d2506d14cc7d47b5f15fa82c and 48b70a9ea5fc37da9accae81f20d1f624ab6e8e6